wagn/rails/controllers/card_controller.rb

```diff
@@ -248,7 +243,9 @@ def show view = nil, status = 200
     else
       args = { :text=>result, :status=>status }
       args[:content_type] = 'text/text' if format == :file
-      render args
+      card.run_callbacks :render_view do
```

Comment:
just noticed that this callback is in the wrong spot. I correct that in a new PR and call it 'show'